api/pkg/apps/webapp/cases.go

```diff
@@ -264,9 +280,7 @@ func (s *Server) Case(w http.ResponseWriter, req *http.Request) {
 		"Comments":         displayComments(comments, commentAuthorMap),
 	}); err != nil {
 		s.Error(w, err)
-		return
 	}
-
 }
```

Comment:
This method is more than 150 lines of code.  I think we can chop it up a little, let's take a pair programming session together.